src/Type/BaseDoctrineORMSerializationType.php

```diff
@@ -146,6 +148,11 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         }
     }
 
+    /**
+     * @psalm-suppress MethodSignatureMustProvideReturnType
```

Comment:
Probably is removed by adding a returntypewillchange attribute. not a big deal since the class is deprecated tho.